Automatic Validation ended with:
> 2025-02-21T21:53:06.9629335Z ##[error] Manifest Error: Duplicate installer entry found.
 2025-02-21T21:53:06.9639397Z ##[error] Manifest Error: Duplicate installer entry found.

(Automated response - build 981.)

--- a/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
+++ b/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
@@ -7,7 +7,7 @@ InstallerType: zip
 NestedInstallerType: portable
 ReleaseDate: 2025-02-21
 Installers:
-- Architecture: neutral
+- Architecture: arm64
   NestedInstallerFiles:
   - RelativeFilePath: ARM64/nefconc.exe
     PortableCommandAlias: nefconc

--- a/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
+++ b/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
@@ -15,7 +15,7 @@ Installers:
     PortableCommandAlias: nefconw
   InstallerUrl: https://github.com/nefarius/nefcon/releases/download/v1.12.0/nefcon_v1.12.0.zip
   InstallerSha256: 0A1597ACAA54991918016523A598F1D820A098CEF633E4A2DC2E49B1CB3DF5AA
-- Architecture: neutral
+- Architecture: x64
   NestedInstallerFiles:
   - RelativeFilePath: x64/nefconc.exe
     PortableCommandAlias: nefconc

--- a/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
+++ b/manifests/n/Nefarius/nefcon/1.12.0/Nefarius.nefcon.installer.yaml
@@ -23,7 +23,7 @@ Installers:
     PortableCommandAlias: nefconw
   InstallerUrl: https://github.com/nefarius/nefcon/releases/download/v1.12.0/nefcon_v1.12.0.zip
   InstallerSha256: 0A1597ACAA54991918016523A598F1D820A098CEF633E4A2DC2E49B1CB3DF5AA
-- Architecture: neutral
+- Architecture: x86
   NestedInstallerFiles:
   - RelativeFilePath: x86/nefconc.exe
     PortableCommandAlias: nefconc

